src/main/resources/application.conf

```diff
@@ -152,6 +152,10 @@ grothendieck {
       # Enabled JSON-RPC APIs over the HTTP endpoint
       # Available choices are: eth, web3, net, personal
       apis = "eth,web3,net"
+
+      net {
+        peer-manager-timeout = 5.seconds
```

Comment:
(We're leaving out config that we don't really expect the user to change without having to talk to a developer. i.e. it's legit to leave it out of the universal file under certain conditions)